Volodymyr_Tsisinskyi/Nedo_net/Nedo_net/Controllers/StudentsController.cs

```
@@ -59,7 +54,14 @@ public IActionResult Post([FromBody] Student value)
         {
             try
             {
-                return Ok();
+                if (ModelState.IsValid)
```

Are you sure that you should throw an exception if ModelState isValid?

---

I checked it in last commit